### DISCUSSION TITLE
Can I reduce the number of jobs `zppy` launches?

### DISCUSSION THREAD

**Discussion by forsyth2** [E3SM-Project/zppy#292] [Questions] [answered]
Can I reduce the number of jobs `zppy` launches?

**Comment by forsyth2** [comment, marked as answer]
Yes, the job bundling option introduced in #243 can be used. For an example configuration file, see https://github.com/E3SM-Project/zppy/blob/main/tests/integration/template_bundles.cfg.

Notable:
- You can use `[bundle]` to specify bundle-specific option.
- You can add a task to a bundle with `bundle = "<bundle-name>"`.
- Subtasks can be placed in bundles other than the default from their parent task. (e.g., `bundle = "bundle2" # Override bundle1`)

**Reply by cameronsmith1** [reply]
Can I use _bundle_ at the top of the configuration file so everything will be in a single script?

**Reply by forsyth2** [reply]
It's not specifically tested in our test (https://github.com/E3SM-Project/zppy/blob/main/tests/integration/template_bundles.cfg), but I'm almost certain that it works. E.g., set `bundle = "one_bundle"` in the `[default]` section.
